packages/blade/public/universal/types.ts

```diff
@@ -31,5 +31,6 @@ export type {
 
 export type * from 'blade-client';
 export type * from 'blade-client/types';
-export type { ResultRecord } from 'blade-compiler';
-export type { ReducedFunction } from 'blade-syntax/queries';
+
+export type ReducedFunction = any;
+export type ResultRecord = any;
```

Comment:
I have no clue why these are causing a build error, unfortunately, so I'm temporarily setting them to `any`. I know that codegen likely relies on them, but I need to unblock myself on https://github.com/ronin-co/blade/pull/469 and codegen likely won't break from `any`.